pages/perfil/index.public.js

```diff
@@ -197,13 +207,22 @@ function EditProfileForm() {
             aria-label="Seu nome de usuário"
             contrast
             sx={{ px: 2, '&:focus-within': { backgroundColor: 'canvas.default' } }}
+            onChange={handleUsernameChange}
           />
-          {errorObject?.key === 'username' && (
-            <FormControl.Validation variant="error">{errorObject.message}</FormControl.Validation>
+          {showUsernameCaption && (
+            <FormControl.Caption>
+              Alterar o nome de usuário irá quebrar todas as URLs das suas publicações e comentários.
```

Comment:
Para a mensagem ficar mais correta:

```suggestion
              Alterar o nome de usuário pode quebrar todas as URLs das suas publicações e comentários.
```